configs/configgen.py

```diff
@@ -139,5 +139,5 @@ def generate_config(template_path, template, output_file, **context):
                 mongos_servers=mongos_servers)
 
 shutil.copy(os.path.join(SCRIPT_DIR, 'envoyproxy_io_proxy.yaml'), OUT_DIR)
-shutil.copy(os.path.join(SCRIPT_DIR, 'encapsulate_in_connect.yaml'), OUT_DIR)
+shutil.copy(os.path.join(SCRIPT_DIR, 'encapsulate_in_http2_connect.yaml'), OUT_DIR)
```

Comment:
Should the new HTTP1 example be included here?

Reply:
not sure about this, WDYT @alyssawilk ?

Reply:
Yeah, might as well for completeness.

Reply:
done